pkg/sriov/sriov.go

```diff
@@ -65,6 +65,8 @@ func (s *sriovManager) SetupVF(conf *sriovtypes.NetConf, podifName string, netns
 		return "", fmt.Errorf("error getting VF netdevice with name %s", linkName)
 	}
 
+	conf.OrigVfState.AllMulti = linkObj.Attrs().Allmulti != 0
```

Comment:
can you move this into allMulti block at L112 ?

Reply:
sure